ci-operator/config/redhat-appstudio/infra-deployments/redhat-appstudio-infra-deployments-main.yaml

```diff
@@ -29,16 +29,16 @@ tests:
     - as: appstudio-e2e-deployment
       cli: latest
       commands: |
-        export PATH=$PATH:/tmp/bin
```

Comment:
What's the reason for removing kustomize installation?

Reply:
kustomize installation was present from the first day and right now kustomize it is not needed anymore